lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -1010,6 +1010,152 @@ module Resources
+          'system': 'urn:oid:2.16.840.1.113883.3.42.10001.100001.12',
+          'value': '1000000105'
+        }
+      ],
+      'active': true,
+      'patient': {
+        'reference': 'Patient/12724068',
+        'display': 'SMART, HAILEY'
+      },
+      'relationship': [
+        {
+          'id': 'CI-490017041-0',
+          'extension': [
+            {
+              'valuePeriod': {
+                'start': '2019-12-26T16:06:26.000Z'
+              },
+              'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/period'
+            },
+            {
+              'valueCodeableConcept': {
+                'coding': [
+                  {
+                    'system': 'http://terminology.hl7.org/CodeSystem/v3-RoleCode',
+                    'code': 'MTH',
+                    'display': 'mother',
+                    'userSelected': false
+                  }
+                ],
+                'text': 'Mother'
+              },
+              'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/relation'
+            }
+          ],
+          'coding': [
+            {
+              'system': 'http://terminology.hl7.org/CodeSystem/v3-RoleCode',
+              'code': 'GT',
+              'display': 'Guarantor',
+              'userSelected': false
+            }
+          ],
+          'text': 'Default Guarantor'
+        }
+      ],
+      'name': [
+        {
+          'id': 'CI-12724066-0',
+          'use': 'official',
+          'text': 'SMART, NANCY',
+          'family': 'SMART',
+          'given': [
+            'NANCY'
+          ],
+          'period': {
+            'start': '2019-12-26T15:12:34.000Z'
+          }
+        }
+      ],
+      'telecom': [
+        {
+          'id': 'CI-PH-29852801-0',
+          'extension': [
+            {
+              'valueUrl': '13215555555',
+              'url': 'http://hl7.org/fhir/StructureDefinition/iso21090-TEL-address'
+            }
+          ],
+          'system': 'phone',
+          'value': '13215555555',
+          'use': 'work',
+          'period': {
+            'start': '2021-03-22T14:12:37.000Z'
+          }
+        }
+      ],
+      'gender': 'female',
+      'birthDate': '1980-08-11',
+      'address': [
+        {
+          'id': 'CI-24313551-0',
+          'use': 'home',
+          'text': '3P\\nKANSAS CITY, MO 641510001\\nUS',
+          'line': [
+            '3P'
+          ],
+          'city': 'KANSAS CITY',
+          'district': 'Jackson',
+          'state': 'MO',
+          'postalCode': '641510001',
+          'country': 'US',
+          'period': {
+            'start': '2020-12-08T15:05:44.000Z'
+          }
+        }
+      ],
+      'communication': [
+        {
+          'language': {
+            'coding': [
+              {
+                'system': 'urn:ietf:bcp:47',
+                'code': 'en',
+                'display': 'English',
+                'userSelected': false
+              }
+            ],
```

Comment:
This part remains unmodified.

Reply:
Added it back https://github.com/cerner/fhir.cerner.com/pull/689/commits/6daf619d1ea7a63aa2371b5a83b35be522920031#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR1144-R1149